### PR TITLE
Open-source Wechaty in Python, Go, and Java

### PR DESCRIPTION
**Pull request by gangena**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

File: jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+### Java Wechaty <https://github.com/wechaty/java-wechaty>
+
+![java](/assets/2020/04-open-source-wechaty-in-python-go-and-java/java.webp)
+
+若感兴趣，还有更多：<https://wechaty.js.org/blog/>
+
+最后，欢迎大家加入wechaty的社区，项目、文档、博客的地址。
+
+项目：<https://github.com/wechaty/wechaty>
+文档：<https://wechaty.js.org/v/zh/>
```
Comment (lijiarui) [inline jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md]: Please change this to http://wechaty.js.org/

**Review by lijiarui — CHANGES_REQUESTED** [review]
LGTM

File: jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md (hunk 1 above)
Comment (gangena) [reply jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md]: It has been changed.

**Comment by gangena** [comment]
Corresponding blog herehttps://github.com/wechaty/wechaty.js.org/issues/909

**Comment by gangena** [comment]
Corresponding blog here<https://github.com/wechaty/wechaty.js.org/issues/909>

File: jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+title: "Open-source Wechaty in Python, Go, and Java"
+author: juzibot
+categories: announcement
+tags:
+  - summer-of-wechaty
+  - summer-2021
```
Comment (lijiarui) [inline jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md]: Please remove the useless tag here.

And add more meaningful tag, such as python, go, java

File: jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+![li](/assets/2020/04-open-source-wechaty-in-python-go-and-java/li.webp)
+
+## 直播纪实
+
+{% include iframe.html src="https://youtu.be/ZX_Pb9fRwS4?list=PL8hd9KDTdarDXf_Rxtr8meKhxtgcXMInh" %}
+
+### Agendas议程
+
+- [0:08:00](https://www.youtube.com/watch?v=ZX_Pb9fRwS4&list=PL8hd9KDTdarDXf_Rxtr8meKhxtgcXMInh&index=8&t=480s)  Introduction & Warm-up 李卓桓，@huan
+
```
Comment (lijiarui) [inline jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md:29]: Please remove redundant lines here.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md (hunk 3 above)
Comment (gangena) [reply jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md:29]: It's all changed.

File: jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md (hunk 2 above)
Comment (lijiarui) [reply jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md]: How about this? @gangena

Comment (gangena) [reply jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md]: It's also changed.

Comment (lijiarui) [reply jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md]: It didn't change, please check for this more carefully

File: jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md (hunk 3 above)
Comment (lijiarui) [reply jekyll/_posts/2020-04-20-open-source-wechaty-in-python-go-and-java.md:29]: No, you haven't change yet.

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by huan** [comment]
Close because the post has already been published.

**Pull request closed without merging**